docs/examples/apim.md

```diff
@@ -111,6 +111,54 @@ provider:
       exposeHeaders:
         - "*"
 
+    # JWT validation APIM policy
+    jwtValidate:
+      headerName: authorization
+      scheme: bearer
+      failedStatusCode: 401
+      failedErrorMessage: "Authorization token is missing or invalid"
+      openId:
+        metadataUrl: "https://path/to/openid/metadata/config"
+      audiences:
```

Comment:
pretty cool - so standard signature validation is included aside from custom audience/issuer?

Reply:
Yes, it supports a variety of configurations.  You can review the APIM docs @ https://docs.microsoft.com/en-us/azure/api-management/api-management-access-restriction-policies#ValidateJWT